### PR TITLE
Us28 profil page

### PR DESCRIPTION
**Pull request by Adwarf14**

### CODE REVIEW

**Comment by LoicBrassart** [comment]
Please stop asking for my reviews, the first one should always be the SM's :pray:

**Review by Gumatan — DISMISSED** [review]

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
`.env` on GH, I won't read the rest of this PR.

**Comment by ghost** [comment]
> `.env` on GH, I won't read the rest of this PR.

.env deletd

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/EditProfile/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+const Profile = () => {
+  const [file, setFile] = useState("https://via.placeholder.com/250");
+  const [imagePreviewUrl, setImagePreviewUrl] = useState(
+    "https://via.placeholder.com/250"
+  );
+
+  const handleImageChange = e => {
+    e.preventDefault();
+    console.log(e.target.files);
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/Profile.jsx]: Nope!

File: frontend/src/components/EditProfile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    reader.readAsDataURL(selectedFile);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    // post of profile picture
+    let data = new FormData();
+    data.append("monfichier", file);
+    axios.post("http://localhost:5050/imgupload", data).then(response => {
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/Profile.jsx]: Stop using `localhost`

File: frontend/src/components/EditProfile/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      alert("image added with success");
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/Profile.jsx]: I think this `alert` really breaks UX, and I'd prefer a quiet notification (if you really *need* to inform your user)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please clean your code (nothing broken, just instructions I don't want to see on `dev` and things that *will* break when in production)

**Review by Gumatan — APPROVED** [review]

File: frontend/src/components/EditProfile/Profile.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import "../style/EditProfileStyles/EditProfile.scss";
+import React, { useState } from "react";
+import axios from "axios";
+import { backend } from "../conf.js";
+
+const Profile = () => {
+  const [file, setFile] = useState("https://via.placeholder.com/250");
+  const [imagePreviewUrl, setImagePreviewUrl] = useState(
+    "https://via.placeholder.com/250"
```
Comment (LoicBrassart) [inline frontend/src/components/EditProfile/Profile.jsx:9]: (Please stop adding placeholders !)

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**